src/widget/core.rs

```diff
@@ -560,3 +562,17 @@ impl Pod {
         self.state.flags.contains(PodFlags::REQUEST_LAYOUT)
     }
 }
+
+impl Element for Pod {
```

Comment:
Gah: Pods, Elements, Widgets, Views. I'm still struggling to work out / keep track of what each of these is and how they relate to each other. Some kind of glossary / explainer of the key nouns / concepts in Xilem would be incredibly helpful.

Reply:
I agree that extra documentation for this would be beneficial.
Xilem's architecture won't be immediately intuitive to all first learning it, so the more documentation, the better.